meta-resin-common/recipes-core/images/resin-image.bb

```diff
@@ -45,6 +45,8 @@ generate_hostos_version () {
     echo "${HOSTOS_VERSION}" > ${DEPLOY_DIR_IMAGE}/VERSION_HOSTOS
 }
 
+DEPENDS += "jq-native"
```

Comment:
Do you know what exactly triggers this dependency?

Reply:
init_config_json in image-resin.bbclass uses jq to parse the board json file.